examples/core/auth-nextjs-pages/src/data/orders.ts

```diff
@@ -0,0 +1,200 @@
+'use client';
```

Comment:
Would it make sense to keep this example about authentication and next.js, and create a new example for data sources?

Reply:
It would if we're going for many small examples instead of just a few with multiple things.
As we already have quite a few examples this seems to be the approach we're following, so I guess 3 new examples would be okay, I'll change it.